src/sources/socket/udp.rs

```diff
@@ -46,7 +46,12 @@ pub fn udp(
     shutdown: ShutdownSignal,
     out: Pipeline,
 ) -> Source {
-    let mut out = out.sink_map_err(|e| error!("error sending event: {:?}", e));
+    let mut out = out.sink_map_err(|error| {
+        emit!(SocketSendError {
```

Comment:
I don't think this is actually a socket error. This would occur when we can't forward the new event into the output pipeline/channel.

Reply:
So this should have remained a plain error and not an event then?

Reply:
Yes. If it does occur it will be because the whole topology is shutting down, so not really the same thing as a socket error and not something I'm too worried about tracking.